src/View/layout.html.twig

```diff
@@ -9,10 +9,10 @@
     <title>Quizzy - le plus grand quiz du monde !{% block title %}{% endblock %}</title>
 
     <link rel="stylesheet" href="/assets/css/style.css">
-    <link rel="stylesheet" href="/assets/css/formDisplayQuestion.css">
     {% block stylesheet %}{% endblock %}
     <link rel="icon" href="/assets/images/favicon.png">
     <script src="https://kit.fontawesome.com/ab41dec404.js" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/canvas-confetti@1.4.0/dist/confetti.browser.min.js"></script>
```

Comment:
You could consider to host you assets locally.
One of the benefit would be to have your site working offline